tests/serialized_test_data_generation/submit_job.sh

```diff
@@ -1,12 +1,37 @@
 #!/bin/bash
 
+# stop on all errors
+set -e
+
+# setup run directory
 RUNDIR=/rundir
+cp /FV3/fv3.exe $RUNDIR/fv3.exe
+
+# setup directory for serialized data
 TEST_DATA_DIR=$RUNDIR/test_data
+mkdir -p $TEST_DATA_DIR
+
+# setup environment
+
+# run the model (deduce MPI configuration from config file)
+cd $RUNDIR
 NUM_RANKS=`python ${RUNDIR}/count_ranks.py $RUNDIR/fv3config.yml`
 ulimit -s unlimited
-cp /FV3/fv3.exe $RUNDIR/fv3.exe
-mkdir -p $TEST_DATA_DIR
+echo ">>> Running model, inspect stdout.out / stderr.out in rundir for details"
+mpirun -l -np $NUM_RANKS $RUNDIR/fv3.exe 2>$RUNDIR/stderr.out 1>$RUNDIR/stdout.out
```

Comment:
What does -l do?

Reply:
It labels the output with the rank number from which the output originates, e.g. if all ranks write(*,*) 'hello' it would result in
```
[0] hello
[1] hello
[2] hello
[3] hello
...
```